### PR TITLE
code-minimap 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by charlesbjohnson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

[`code-minimap`](https://github.com/wfxr/code-minimap) is a tool for generating text minimaps from source files for use in terminal-based text editors, the same idea as the [minimap in VS Code](https://code.visualstudio.com/docs/getstarted/userinterface#_minimap).

### CODE REVIEW

File: Formula/code-minimap.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    system "cargo", "install", *std_cargo_args
+
+    bash_completion.install "completions/bash/code-minimap.bash"
+    fish_completion.install "completions/fish/code-minimap.fish"
+    zsh_completion.install  "completions/zsh/_code-minimap"
+  end
+
+  test do
+    (testpath/"test.txt").write("hello world")
+    assert_equal "⠉⠉⠉⠉⠉⠁\n", shell_output("#{bin}/code-minimap #{testpath/"test.txt"}")
```
Comment (carlocab) [inline Formula/code-minimap.rb:20]:
```suggestion
    assert_equal "⠉⠉⠉⠉⠉⠁\n", shell_output("#{bin}/code-minimap #{testpath}/test.txt")
```
Let's make this a bit easier to read. Squash this change with your existing commit, please.

**Review by carlocab — COMMENTED** [review]
Thanks, @charlesbjohnson. Just one suggestion.

File: Formula/code-minimap.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    system "cargo", "install", *std_cargo_args
+
+    bash_completion.install "completions/bash/code-minimap.bash"
+    fish_completion.install "completions/fish/code-minimap.fish"
+    zsh_completion.install  "completions/zsh/_code-minimap"
+  end
+
+  test do
+    (testpath/"test.txt").write("hello world")
+    assert_equal "⠉⠉⠉⠉⠉⠁\n", shell_output("#{bin}/code-minimap #{testpath}/test.txt}")
```
Comment (carlocab) [inline Formula/code-minimap.rb:20]:
```suggestion
    assert_equal "⠉⠉⠉⠉⠉⠁\n", shell_output("#{bin}/code-minimap #{testpath}/test.txt")
```
There's a leftover extra `}` here.

Comment (charlesbjohnson) [reply Formula/code-minimap.rb]: 😳 D'oh. Thanks!

**Review by carlocab — APPROVED** [review]
Thanks, @charlesbjohnson.

**Pull request closed without merging**